README.md

```diff
@@ -1,53 +1,99 @@
 <h1 align="center">Marker</h1>
 <p align="center">
-    <br/>
-    <img height=200 src="res/logo.svg" alt="Marker logo"/>
+    <img height=200 src="./res/marker-logo-round.svg" alt="Marker logo" />
 </p>
 
 <br/>
 
-[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/46214)
-[![CI Status](https://github.com/rust-marker/marker/actions/workflows/rust_bors.yml/badge.svg)](https://github.com/rust-marker/marker/actions?query=event%3Apush+workflow%3A%22Rust+(bors)%22)
+[![Crates.io](https://img.shields.io/crates/v/marker_api.svg)](https://crates.io/crates/marker_api)
+<!--
+FIXME(xFrednet): Add license shield, once crates.io also says:
+[![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/marker_api.svg)](#license)
+-->
 
-**An experimental linting interface for Rust. Let's make custom lints a reality!**
+Marker is an experimental code analysis interface, with the goal to create a stable and user-friendly linting framework for the Rust programming language. Creating custom lints should be a straightforward process, and using them should be as simple as declaring a new dependency.
+
+Let's make custom lints and code analysis a reality!
 
 > **Note**
 >
-> The project is in the very early stages of development, some things are still missing and the API is not stable yet.
+> The project is in the early stages of development, some things are still missing and the API is not stable yet.
+>
+> A collection of limitations, goals and features is available below.
+
+## Features
+
+* **Custom Lints**: Marker provides a framework for everyone, to create and provide custom lints. Allowing you to automatically improve the code quality for your project and users of your crate.
```

Comment:
Marker offers a framework for everyone to create and provide custom lints, allowing you to automatically improve the code quality for your project.

Reply:
I want to keep the point, that lints can be provided for other users, highlighted. I like the rewrite of the first part :+1: Now I've written:

* **Custom Lints**: Marker offers a framework for everyone to create and provide custom lints, allowing you to automatically improve the code quality for you and users of your crate.